packages/nuxt/src/core/schema.ts

```diff
@@ -23,11 +23,9 @@ export default defineNuxtModule({
     const resolver = createResolver(import.meta.url)
 
     // Initialize untyped/jiti loader
-    const _resolveSchema = jiti(dirname(fileURLToPath(import.meta.url)), {
-      esmResolve: true,
+    const _resolveSchema = createJiti(dirname(fileURLToPath(import.meta.url)), {
```

Comment:
Perf: jiti accepts a filename and if input is dir, it tries to make a fake path `{dir}/_index.js` itself. Also with `file://` URLs, it skips all these!

```suggestion
    const _resolveSchema = createJiti(import.meta.url), {
```